<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>protocols</class>
 <widget class="QWidget" name="protocols">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Protocol designer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of cells</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_cells">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>901</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current protocol </string>
   </property>
  </widget>
  <widget class="QPushButton" name="addStage">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Stage</string>
   </property>
  </widget>
  <widget class="QLabel" name="ncells">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>n cells</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>83</y>
     <width>909</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="11">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Repeats</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Groups period 
 (ms) 
</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Light ON 
 time (ms)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_10">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add jitter to the stimulation sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Jitter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QLineEdit" name="stim_time">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="groupsNumbers">
      <property name="toolTip">
       <string>When &quot;Squares&quot; type, number of simultanous presented squares</string>
      </property>
      <property name="text">
       <string>Number of 
 groups 
 or squares</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLineEdit" name="groups_period">
      <property name="text">
       <string>400</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="stim_type">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string extracomment="Repeat the whole sequence randomly in each iteration">Random</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="Stimulating only the groups (either manual or auto)">Group only stim.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="Repeat the sequence in the same order">Order</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="Calcium imaging during sequnce run">Test</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="Stimulating randomly all neurons in culture">Spontaneous</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Squares</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QCheckBox" name="jitter">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Background 
 stimulation (Hz)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Probability 
 stimulation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QLineEdit" name="repeats">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Stage</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="on_time">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QLineEdit" name="groups_number">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="prob_stim">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="background_freq">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLineEdit" name="group_size">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Stimulation 
 time (min)</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Is manual 
 groups</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QCheckBox" name="is_manual_sequence">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Group size 
 Square size</string>
      </property>
     </widget>
    </item>
    <item row="2" column="12">
     <widget class="QPushButton" name="output_group">
      <property name="text">
       <string>Output group</string>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Press to define 
 output group</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveProtocol">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(34, 34, 34);
	background-color: rgb(162, 238, 255);
	border-color: rgb(255, 85, 0);
	border-radius: 12
}
QPushButton:
{
	background-color: Blue
}</string>
   </property>
   <property name="text">
    <string>Save Protocol</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeStage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeWindow">
   <property name="geometry">
    <rect>
     <x>814</x>
     <y>540</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close window</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>After change / update add new fields to protocol_design</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
